Documentation/toc.yml

```diff
@@ -97,20 +99,32 @@
         href: Diagnostics/ConfiguringDiagnostics.md
       - name: Using the Visual Profiler
         href: Diagnostics/UsingVisualProfiler.md
-  - name: Other Features
+    - name: Extension Service Creation Wizard
+      href: Utilities/ExtensionServiceCreationWizard.md
+  - name: Experimental Features
     items:
+    - name: Spatial Understanding
+      href: TODO.md
     - name: QR Tracking
       href: TODO.md
     - name: Spatial Audio
       href: TODO.md
   - name: Advanced topics
     items:
-    - name: Performance
-      href: Performance/PerfGettingStarted.md
     - name: Shared Experiences
       href: TODO.md
-    - name: Moving your project from HTK to MRTK
-      href: HTKToMRTKPortingGuide.md
+- name: How-to Guides 
```

Comment:
suggestion: the top level navigation already says guides / maybe just call this section How-tos or How to.... and then continue the children with eg setup global voice commands , trigger an action using MRTK button
that will make them shorter and we don't have the repetition of how to all the time?

Reply:
done